msteams-platform/graph-api/activity-feed/feed-notifications.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Activity Feed notifications in Teams
+description: Describes activity feed notifications in Teams and how to use to the best advantage
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: Overview
+keywords: teams notifications feed activity Graph
+---
+# Microsoft Teams activity feed notifications  — Developer Preview
+
+>[!NOTE]
+>Microsoft Teams Graph API activity feed notifications are currently available in desktop and Android clients after Developer Preview has been enabled. See [How do I enable Developer Preview](../../resources/dev-preview/developer-preview-intro.md) for more information.
+
+The Activity feed API is a Microsoft Teams and Microsoft Graph integration that enables your app to use API endpoints to notify  users of events ranging from tasks requiring a user's attention, to actions taken by others that affect the user or a set of users.
+
+## Activity feed senders and recipients
+
+There are two core concepts for sending notifications:
+
+- **User context**: These notifications are sent on behalf of the user by the application based on a user-initiated activity.
+
+- **Application context**: These notifications are sent directly by the application to users.
+
+There are three core notification recipient types:
+
+- **User**: An individual Teams user.
+
+- **Chat members**: All members in a chat conversation.
+
+- **Channel members**: All members of a team/channel.
+
+> [!NOTE]
+> The Teams Activity Feed notification context is on a per user basis. Although an event may impact an entire team or members of a chat, activity feed notifications, triggered by that event, are sent to individual recipients (users).
+
+## Activity feed actions
+
+|Notification type| Description|URI binding parameter|
+|------------------|-----------------|----------------|-------------|
+|sendActivityNotification| POST a notification to the activity feed of a user. | Azure Active Directory user ID |
+||Sample request| **POST** ht<span>tps://graph.microsoft.com/beta/teams/{userId}/sendActivityNotification |
```

Comment:
Please check the other comment - it should include the 'teamwork' keyword in the URI.